src/store/plugins/names.js

```diff
@@ -25,8 +25,9 @@ export default store =>
       set(state, names) {
         state.owned = names;
       },
-      setDefault({ defaults }, { address, networkId, name }) {
-        Vue.set(defaults, `${address}-${networkId}`, name);
+      setDefault(state, { address, networkId, name }) {
+        const nameDefault = { [`${address}-${networkId}`]: name };
+        state.defaults = { ...(state.defaults || {}), ...nameDefault };
```

Comment:
the previous version was clearer, why do we need this change?

Reply:
for some reason first time when app is opened `defaults` props is `undefined` and `Vue.set(defaults...)` throws an error. When reopen this problem is gone

Reply:
Well, I propose to investigate this issue instead of making a workaround

Reply:
After making `update-default-name.js` migration run in sync and remove usage of `commit` this problem is not happening anymore. Returning this mutation to previous version